src/test/java/com/google/api/generator/engine/ast/ExprStatementTest.java

```diff
@@ -85,6 +93,15 @@ public void invalidExprStatement_value() {
     assertInvalidExprStatement(valueExpr);
   }
 
+  @Test
+  public void invalidExprStatement_logicalNotUnaryOperator() {
+    Expr logicalNotExpr =
+        UnaryOperationExpr.logicalNotWithExpr(
+            VariableExpr.withVariable(
+                Variable.builder().setType(TypeNode.BOOLEAN).setName("foo").build()));
+    assertInvalidExprStatement_(logicalNotExpr);
```

Comment:
nit: I guess accidentally added "_" at the end of `assertInvalidExprStatement`

Reply:
Fixed master merge.